src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/MontaguSerializer.kt

```diff
@@ -23,6 +23,21 @@ interface Serializer
     val gson: Gson
 }
 
+class NullToEmptyStringSerializer(serializer: Serializer = MontaguSerializer.instance): Serializer by serializer {
+
+    override fun serializeValueForCSV(value: Any?) = when (value)
+    {
+        null -> ""
```

Comment:
Could refactor the base class so you only have to override the `null` branch of the when, rather than the whole thing. e.g. by introducing a `serializeNull` method on the base class and then just overriding that in this class?